audiomentations/augmentations/wind_absorption.py

```diff
@@ -0,0 +1,167 @@
+        :param n_fft: The number of bins in the estimated filterbank
+        :param p: The probability of applying this transform
+        """
+        assert float(min_temperature) in [
+            10.0,
+            20.0,
+        ], "Sorry, the only supported temperatures are either 10 or 20 degrees celcius"
+        assert float(max_temperature) in [
+            10.0,
+            20.0,
+        ], "Sorry, the only supported temperatures are either 10 or 20 degrees celcius"
+        assert min_temperature <= max_temperature
+        assert 30 <= min_humidity <= max_humidity <= 90
+
+        super().__init__(p)
+
+        self.min_temperature = min_temperature
+        self.max_temperature = max_temperature
+
+        self.min_humidity = min_humidity
+        self.max_humidity = max_humidity
+
+        self.min_distance = min_distance
+        self.max_distance = max_distance
+
+        self.n_fft = n_fft
+
+    def randomize_parameters(self, samples, sample_rate):
+        super().randomize_parameters(samples, sample_rate)
+        self.parameters["temperature"] = 10 * np.random.randint(
+            int(self.min_temperature) // 10, int(self.max_temperature) // 10 + 1
+        )
+        self.parameters["humidity"] = np.random.randint(
+            self.min_humidity, self.max_humidity + 1
+        )
+        self.parameters["distance"] = np.random.uniform(
+            self.min_distance, self.max_distance
+        )
+
+    def apply(self, samples, sample_rate):
+        assert samples.dtype == np.float32
+
+        humidity = self.parameters["humidity"]
+        distance = self.parameters["distance"]
+
+        # Choose correct absorption coefficients
+        key = str(int(self.parameters["temperature"])) + "C"
+        bounds = [30, 50, 70, 90]
+        for n in range(1, len(bounds)):
+            if bounds[n - 1] <= humidity or humidity <= bounds[n]:
+                key += f"_{bounds[n-1]}-{bounds[n]}%"
+                break
+
+        # Convert to attenuations
+        attenuation_values = np.exp(
+            -distance * np.array(self.air_absorption_table[key])
+        )
+
+        # Frequencies to calculate the attenuations caused by wind absorption
+        frequencies = librosa.fft_frequencies(sr=48000, n_fft=256)
+
+        # Interpolate to the desired frequencies (we have to do this in dB)
+        db_target_attenuations = np.interp(
+            frequencies,
+            self.air_absorption_table["center_freqs"],
+            20 * np.log10(attenuation_values),
+        )
+
+        linear_target_attenuations = 10 ** (db_target_attenuations / 20)
+
+        result = np.zeros_like(samples)
+
+        # Apply using STFT
+        if len(samples.shape) == 1:
+            stft = librosa.stft(samples, n_fft=self.n_fft)
+
+            # Compute mask
+            mask = np.tile(linear_target_attenuations, (stft.shape[1], 1)).T
+
+            # Compute target degraded audio
+            degraded_audio = librosa.istft(stft * mask)
```

Comment:
check if it's possible to specify a desired length to the istft function instead of doing the min_length trick